Makefile

```diff
@@ -404,7 +402,8 @@ test-ci-js: | clear-stalled
 	fi
 
 test-ci: LOGLEVEL := info
-test-ci: | clear-stalled build-addons build-addons-napi doc-only
+test-ci: | clear-stalled test/addons/.buildstamp \
```

Comment:
Now come to think of it, why doesn't test-ci depends on `$(NODE_EXE)`? Or depend on `build-ci`? AFAICT this should work as well

```
test-ci: | clear-stalled build-ci test/addons/.buildstamp test/addons-napi/.buildstamp doc-only

run-ci: test-ci
```

cc @nodejs/build

Reply:
Wouldn't that mean `test-ci` will always run `build-ci` first? I would like `test-ci` to depend only on the things it actually needs, so we can run tests on release tarballs etc.